contracts/src/ethereum/TokenBridge.sol

```diff
@@ -136,4 +140,28 @@ contract TokenBridge is
         require(l1Token.balanceOf(msg.sender) - amount <= l1Token.balanceOf(msg.sender), "UNDERFLOW");
         l1Token.transfer(recipient, amount);
     }
+
+     function consumeBridgeRewardMessage(address l1Token, address recipient, uint256 amount) internal {
+        emit LogBridgeReward(l1Token, recipient, amount);
+
+        uint256[] memory payload = new uint256[](5);
+        payload[0] = BRIDGE_REWARD_MESSAGE;
+        payload[1] = uint256(l1Token);
+        payload[1] = uint256(recipient);
```

Comment:
typo: `payload[2]`instead of `payload[1]`. then increment indices of following lines

Reply:
This change will be overriden (fixed) when merged with your changes to token bridge

Reply:
Not sure it will be fixed as this function is totally new, and won't be merged to a previous implementation.

Reply:
I think we should keep it as a separate function but I am not super hell bent on it. In the case we merge it with consumeMessage we'll have to pass in the value for payload[0] and do an if for emitting the right event.
Do you think we should merge it?